tsconfig.json

```diff
@@ -23,7 +23,7 @@
     /* Modules */
     "module": "commonjs", /* Specify what module code is generated. */
     // "rootDir": "./",                                  /* Specify the root folder within your source files. */
-    "moduleResolution": "node", /* Specify how TypeScript looks up a file from a given module specifier. */
+    "moduleResolution": "node16", /* Specify how TypeScript looks up a file from a given module specifier. */
```

Comment:
I imagine this change is so TS doesn't complain about top-level await, among other things mentioned in https://github.com/microsoft/TypeScript/pull/48879?

Reply:
`node16` also enables support for Typescript following `package.json`'s `exports` field, which hydrogen adopted for v1.

Reply:
Yes, this is so we can import from `@netlify/hydrogen-platform/handler`, for example.